client/state/terms/actions.js

```diff
@@ -35,7 +35,7 @@ export function addTerm( siteId, taxonomy, term ) {
 		} ) );
 
 		return wpcom.site( siteId ).taxonomy( taxonomy ).term().add( term ).then(
-			( data ) => dispatch( receiveTerm( siteId, taxonomy, omit( data, '_headers' ) ) ),
+			( data ) => dispatch( receiveTerm( siteId, taxonomy, omit( data, '_headers' ) ), null, null ),
```

Comment:
What's `null, null` for? `dispatch` only [takes a single argument](http://redux.js.org/docs/api/Store.html#dispatch)